internal/provider/provider.go

```diff
@@ -222,6 +228,18 @@ func (p *pingfederateProvider) Configure(ctx context.Context, req provider.Confi
 		}
 	}
 
+	var xBypassExternalValidation bool
+	var xBypassExternalValidationErr error
+	if !config.XBypassExternalValidation.IsUnknown() && !config.XBypassExternalValidation.IsNull() {
+		xBypassExternalValidation = config.XBypassExternalValidation.ValueBool()
+	} else {
+		xBypassExternalValidation, xBypassExternalValidationErr = strconv.ParseBool(os.Getenv("X_BYPASS_EXTERNAL_VALIDATION"))
```

Comment:
The environment variable should follow the pattern of the other ones and include the same prefix. It should also be mentioned in the description of the attribute so that it shows up in the docs.